packages/server/src/api/queueRoutes.ts

```diff
@@ -137,3 +118,43 @@ export const queueRoutes: ServerRoute[] = [
     },
   },
 ];
+
+// for some reason, JOI.allow(null) means the property has to exist, but can be null
+
+async function questionModelToQuestion(qm: QuestionModel): Promise<Question> {
+  return {
+    creator: await userModelToUserPartial(await (await qm.creator).user),
+    id: qm.id,
+    createdAt: qm.createdAt,
+    status: qm.status,
+    text: qm.text,
+    // qm.taHelped: types says is nonnullable, but it is nullable
+    taHelped:
+      (await qm.taHelped) &&
+      (await userCourseModelToUserPartial(await qm.taHelped)),
```

Comment:
if property `taHelped` exists on the QuestionModel, then convert it to a UserPartial. if not, then leave it as null.

Reply:
this is needed because of the above (taHelped may be null, in which case don't try to convert)